distribution/install.sh

```diff
@@ -207,6 +207,29 @@ add_to_path() {
 # be updated if necessary.
 # ------------------------------------------------------------------------------
 
+get_gnu_musl_glibc() {
+  need_cmd ldd
```

Comment:
I think it might be a bit presumptuous to assume folks have `ldd`? Could we have a safe fallback?

Reply:
We already presume it in the current script so this is status quo.